spec/unit/generator/entry_spec.rb

```diff
@@ -0,0 +1,351 @@
+# frozen_string_literal: true
+
+describe GitHubChangelogGenerator::Entry do
+  def label(name)
+    { "name" => name }
+  end
+
+  def issue(title, labels, number = "1", user = { "login" => "user" })
+    {
+      "title" => "issue #{title}",
+      "labels" => labels.map { |l| label(l) },
+      "number" => number,
+      "html_url" => "https://github.com/owner/repo/issue/#{number}",
+      "user" => user
+    }
+  end
+
+  def pr(title, labels, number = "1", user = { "login" => "user" })
+    {
+      "pull_request" => true,
+      "title" => "pr #{title}",
+      "labels" => labels.map { |l| label(l) },
+      "number" => number,
+      "html_url" => "https://github.com/owner/repo/pull/#{number}",
+      "user" => user.merge("html_url" => "https://github.com/#{user['login']}")
+    }
+  end
+
+  def get_titles(issues)
```

Comment:
Could we change this name to be more about the result of the call?

Example method name:

- `titles_for(issues)`